frontend-react/src/network/QueryClients.ts

```diff
@@ -9,6 +9,8 @@ export const appQueryClient = new QueryClient({
             staleTime: Infinity,
             cacheTime: Infinity,
             refetchOnWindowFocus: false,
+            onError: (error: any) =>
+                console.error(`Something went wrong: ${error.message}`),
```

Comment:
We may want to to create a new error with this message (setting the `cause` property to the original error) and console.error that to keep an intact stack trace.

Reply:
Will make this change.